pkgs/by-name/mi/midisheetmusic/package.nix

```diff
@@ -1,19 +1,36 @@
 { lib, stdenv, fetchurl
-, mono, dotnetPackages, makeWrapper
-, gtk2, cups, timidity }:
+, mono, unzip, runCommand, mkNugetDeps, makeWrapper
+, makeFontsConf, gtk2, cups, timidity }:
 
 let
-  version = "2.6";
-in stdenv.mkDerivation {
+  deps = mkNugetDeps {
+    name = "midisheetmusic-deps";
+    nugetDeps = { fetchNuGet }: [
+      (fetchNuGet {
+        pname = "NUnit.Console";
+        version = "3.0.1";
+        sha256 = "154bqwm2n95syv8nwd67qh8qsv0b0h5zap60sk64z3kd3a9ffi5p";
+      })
+      (fetchNuGet {
+        pname = "NUnit";
+        version = "2.6.4";
+        sha256 = "1acwsm7p93b1hzfb83ia33145x0w6fvdsfjm9xflsisljxpdx35y";
+      })
+    ];
+  };
+  deps-unpacked = runCommand "midisheetmusic-deps-unpacked" {} ''
+    find ${deps} -name '*.nupkg' \
+      -exec sh -c 'mkdir -p $out/$(basename "$1") && ${unzip}/bin/unzip -d $out/$(basename "$1") "$1"' sh {} \;
+  '';
```

Comment:
Can't we use the standard dep file generation tooling?

Reply:
Not in this case, since this is a .NET Framework project, and out tooling doesn't support it (and it doesn't have to, since there's only like 1 or 2 packages using .NET Framework in nixpkgs)